src/ProjectOrigin.WalletSystem.IntegrationTests/TestClassFixtures/JwtTokenIssuerFixture.cs

```diff
@@ -3,27 +3,30 @@
 using System.IO;
 using System.Security.Claims;
 using System.Security.Cryptography;
+using System.Text.Json;
 using AutoFixture;
 using Grpc.Core;
 using Microsoft.IdentityModel.Tokens;
 
 namespace ProjectOrigin.WalletSystem.IntegrationTests.TestClassFixtures;
 
-public class JwtTokenIssuerFixture : IDisposable
+public sealed class JwtTokenIssuerFixture : IDisposable
 {
-    public string Issuer { get; init; } = "TestIssuer";
-    public string Audience { get; } = "WalletSystem";
-    public int ExpirationMinutes { get; } = 60;
-    public string PemFilepath { get; }
+    private readonly RSA _rsa = RSA.Create();
+
+    public string KeyType => "RSA";
+    public string Algorithm => SecurityAlgorithms.RsaSha256;
+    public byte[] PublicKeyInfo => _rsa.ExportSubjectPublicKeyInfo();
+    public int ExpirationMinutes => 60;
+    public string PemFilepath { get; private init; }
+    public string Audience => "WalletSystem";
 
-    private readonly ECDsa _ecdsa;
-    private bool _disposed = false;
+    public string Issuer { get; init; } = "TestIssuer";
 
     public JwtTokenIssuerFixture()
     {
-        _ecdsa = ECDsa.Create(ECCurve.NamedCurves.nistP256);
         PemFilepath = Path.GetTempFileName();
-        var pem = _ecdsa.ExportSubjectPublicKeyInfoPem();
+        var pem = _rsa.ExportSubjectPublicKeyInfoPem();
```

Comment:
We have an "EnergyOrigin.TokenValidation" NuGet package, which we use for testing purposes. Just FYI :)

Reply:
I know, but I think it would hard to lock the open source project on the more local packages for EnergyOrigin.